modules/mongodb/mongodb_test.go

```diff
@@ -76,3 +78,38 @@ func ExampleRunContainer_connect() {
 	// Output:
 	// test
 }
+
+func TestMongoDB(t *testing.T) {
```

Comment:
One last thing before merging this PR. The code in this file is having only testable examples. Are you OK having your test as a regular test or do you prefer to have it as a testable example? If so, you'd need to rename the test method to `ExampleRunContainer_withCredentials`, printing messages and checking them in an output comment. Please see above examples

Reply:
One of the benefits for testable examples is they appear in the pkg.go.dev site